Detectors/GlobalTracking/src/MatchTOF.cxx

```diff
@@ -553,20 +554,21 @@ void MatchTOF::propagateTPCTracks(int sec)
     }
 
     if (trc.getX() < o2::constants::geom::XTPCOuterRef - 1.) {
-      if (!propagateToRefX(trc, o2::constants::geom::XTPCOuterRef, 10, intLT0) || TMath::Abs(trc.getZ()) > Geo::MAXHZTOF) { // we check that the propagat>
+      if (!propagateToRefX(trc, o2::constants::geom::XTPCOuterRef, 10, intLT0dummy) || TMath::Abs(trc.getZ()) > Geo::MAXHZTOF) { // we check that the propagat>
```

Comment:
just remove passing any `intLT...` to this test propagation, the double counting happens here since then this step is repeated anyways.

Reply:
done